test/pathforaSpec.js

```diff
@@ -1369,6 +1369,39 @@ describe('API', function () {
     expect(widget.length).toBe(1);
   });
 
```

Comment:
in the spirit of being thorough it probably also makes sense to add a few tests to ensure that when a cookie is invalid it doesn't blow up pathfora / cause panics. 

when its missing an impression count or a value or is totally blank, etc. since cookies are public there is always a chance that someone will goof our cookie(s)